instaclustr/api_client.go

```diff
@@ -222,3 +222,77 @@ func (c *APIClient) ReadVpcPeering(cdcID string, vpcPeeringID string) (*VPCPeeri
 	json.Unmarshal(bodyText, &vpcPeering)
 	return &vpcPeering, nil
 }
+
+func (c *APIClient) EncryptionKeyAdd(data []byte) (string, error) {
```

Comment:
Could this be renamed to CreateEncryptionKey for consistency with other creation methods? Example -CreateCluster